src/main/resources/application.properties

```
@@ -1 +1,12 @@
+#Database
+db.driver=com.mysql.jdbc.Driver
+db.url=jdbc:mysql://localhost:3306/bookstore?serverTimezone=UTC
+db.username=root
+db.password=1469
```

it's better not to pass sensitibe data and credentials to the open sources
```suggestion
db.username=
db.password=
```